lib/main.js

```diff
@@ -172,83 +164,93 @@ module.exports = class Router extends EventEmitter {
         };
     }
 
-    postBundleParseCallback() {
-        return (req, res, next) => {
-            body(req, res, {}, (error, bodyObj) => {
-                if (error) {
-                    return next(
-                        boom.boomify(error, {
-                            statusCode: 400,
-                            message: 'Failed parsing postBundle POST-body',
-                        })
-                    );
+    postBundleHandler(type) {
+        assert(
```

Comment:
do we not catch the asset exception? Won't the server crash if the `assert` fails? Wouldn't it be better to return 400 bad request to the user? Or is there a global catch assert errors middleware handling this somewhere?

Reply:
This is a developer only error. Just a sanity check to make sure we used the method correctly. Once the server is running there would never be an error here.

Reply:
I was just  a bit trigger-happy and see now that these values are hard-coded and not part of the reques!

> Once the server is running there would __never__ be an error here.

:+1: Famous last words? :cocktail:

Reply:
Famous...last...words. 
So then goodbye cruel world...